person.rb

```diff
@@ -4,7 +4,7 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age, :rentals
+  attr_accessor :name, :age, :rentals, :parent_permission
 
   def initialize(age, name = 'Unknown', parent_permission: true)
```

Comment:
- [ ] When I try to create a teacher I get an error and the app stop, there is the wrong number of arguments. Instead of `parent_permission: true` is `parent_permission = true` and everything will work 😉 🙃 

<img width="1120" alt="Screenshot 2023-03-25 at 14 51 55" src="https://user-images.githubusercontent.com/74501523/227716808-407abe9b-145a-4e90-8991-dc6c6c89b221.png">

Reply:
- [ ] When I change `parent_permission = true ` it gives a linter error!, so instead of that I put its value in '**single quotes**'